### PR TITLE
Refactor:Use PARAM instead of deepks*

### PR DESCRIPTION
**Pull request by A-006**
### Unit Tests and/or Case Tests for my changes
- Change deepks CMakeLists.txt and unit test files.

### What's changed?
- Change the Deepks files in the cpp files,add Conditional judgment in the set globalv file.

### CODE REVIEW

File: source/module_io/read_input_item_deepks.cpp [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #include "module_base/constants.h"
+#include "module_base/global_variable.h"
```
Comment (mohanchen) [inline source/module_io/read_input_item_deepks.cpp]: input depends on global_variable?

File: source/module_io/input_conv.cpp [hunk 2]
```diff
@@ -557,45 +557,12 @@ void Input_Conv::Convert()
...
-        {
-            ModuleBase::WARNING_QUIT("Input_conv", "generate deepks unittest with only 1 processor");
-        }
-        if (PARAM.inp.cal_force != 1)
-        {
-            ModuleBase::WARNING_QUIT("Input_conv", "force is required in generating deepks unittest");
-        }
-        if (GlobalV::CAL_STRESS != 1)
+    #ifndef __DEEPKS
+        if (PARAM.inp.deepks_scf || PARAM.inp.deepks_out_labels || PARAM.inp.deepks_bandgap || PARAM.inp.deepks_v_delta)
```
Comment (Qianruipku) [inline source/module_io/input_conv.cpp]: mv it to check_value

File: source/module_io/read_input.cpp [hunk 3]
```diff
@@ -315,6 +318,7 @@ void ReadInput::read_txt_input(Parameter& param, const std::string& filename)
             resetvalue_item->reset_value(*resetvalue_item, param);
 }
     }
+    this->set_globalv(param);
```
Comment (Qianruipku) [inline source/module_io/read_input.cpp:321]: It is repeated.

File: source/module_parameter/system_parameter.h [hunk 4]
```diff
@@ -31,11 +31,18 @@ struct System_para
...
-    std::string global_in_card = "INPUT";        ///< global input card
+    std::string global_in_card = "INPUT";   ///< global input directory
     std::string global_out_dir = "";        ///< global output directory
     std::string global_readin_dir = "";     ///< global readin directory
     std::string global_stru_dir = "";       ///< global structure directory
     std::string global_matrix_dir = "";     ///< global matrix directory
+
+    bool deepks_setorb = false;             ///< true if "deepks" is set
+    
+    int npol = 1;                           ///< number of polarization
```
Comment (Qianruipku) [inline source/module_parameter/system_parameter.h]: Where to use npol?

**Review by Qianruipku — CHANGES_REQUESTED** [review]

File: source/module_io/read_input_item_deepks.cpp (hunk 1 above)
Comment (A-006) [reply source/module_io/read_input_item_deepks.cpp]: Previously considered adding the GlobalV parameter, but it has been commented out. The dependency can be removed directly here, i will remove it now.

File: source/module_io/input_conv.cpp (hunk 2 above)
Comment (A-006) [reply source/module_io/input_conv.cpp]: The parameter has been move to the check_vlaue in PARAM.inp.deepks_scf.

File: source/module_io/read_input.cpp (hunk 3 above)
Comment (A-006) [reply source/module_io/read_input.cpp:321]: The abundant function has been deleted.

File: source/module_parameter/system_parameter.h (hunk 4 above)
Comment (A-006) [reply source/module_parameter/system_parameter.h]: The npol will use in the next PR,i has deleted the paramter in this PR.

**Review by Qianruipku — APPROVED** [review]

**Pull request merged**